cscs-checks/apps/spark/spark_check.py

```diff
@@ -10,26 +10,22 @@
 from reframe.core.backends import getlauncher
 
 
-@rfm.simple_test
-class SparkCheck(rfm.RunOnlyRegressionTest):
+class SparkBaseCheck(rfm.RunOnlyRegressionTest):
     def __init__(self):
-        self.descr = 'Simple calculation of pi with Spark'
         self.valid_systems = ['daint:gpu', 'daint:mc',
                               'dom:gpu', 'dom:mc']
```

Comment:
Can we add eiger here too?

Reply:
Spark is currently not on eiger's software stack.